### PR TITLE
Add .env.example, .eslintignore, Moved mailgun credentials to .env

### PR DESCRIPTION
**Pull request by sayyidyofa**
Added .env.example
Added .eslintignore to ignore `build` directory
Removed .env because can lead to accidentally revealing production credentials
Moved mailgun credentials to .env

### CODE REVIEW

**Comment by santiq** [comment]
Awesome, thanks!

**Comment by sayyidyofa** [comment]
Hey @santiq I'm really sorry but if you look again, it seems like I've forgot to add the `env.example` file in my PR. Maybe you can make one yourself based on [this commit](https://github.com/sayyidyofa/bulletproof-nodejs/commit/66b43d84c751503ff4a75af045e880a3fe5b6ad6) in my fork because I think one more PR for just a single file is a bit unnecessary

**Comment by santiq** [comment]
@sayyidyofa 
No problem 👍

File: .env [hunk 1]
```diff
@@ -1,13 +0,0 @@
...
-
-# Mongo DB
-# Local development
-MONGODB_URI='mongodb://localhost/bulletproof-nodejs'
-
-# Port
-PORT=3000
-
-# Debug 
-LOG_LEVEL='debug'
```
Comment (arled) [inline .env:13]: Why did you remove this file?!

Comment (sayyidyofa) [reply .env:13]: Because the best practice on developing with `.env` file is: `.env` file is not supposed to be commited into a repository, as it will accidentally reveal production credentials, as per I have mentioned in my PR note
